modules/ruxhal/src/arch/x86_64/mod.rs

```diff
@@ -19,6 +19,8 @@ use core::arch::asm;
 use memory_addr::{PhysAddr, VirtAddr};
 use x86::{controlregs, msr, tlb};
 use x86_64::instructions::interrupts;
+
+#[allow(unused_imports)]
```

Comment:
I think use `#[cfg(feature = "musl")]` would be more clearness than `#[allow(unused_imports)]`.

Reply:
Adjusted.